recipes/hdbscan/meta.yaml

```diff
@@ -0,0 +1,48 @@
+{% set version = "0.8" %}
+
+package:
+  name: hdbscan
+  version: {{ version }}
+
+source:
+  fn: hdbscan-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/h/hdbscan/hdbscan-{{ version }}.tar.gz
+  sha256: ea09e8ad60a1734e46000e71cc4b4ca274629e5ed9c54015cf4d4150c1238a17
+
+build:
+  number: 0
+  script: 
+    # FIXME: This is a hack to make sure the environment is activated.
+    # The reason this is required is due to the conda-build issue
+    # mentioned below.
+    #
+    # https://github.com/conda/conda-build/issues/910
+    #
+    - source activate "${CONDA_DEFAULT_ENV}"  # [unix]
+    - python setup.py install --single-version-externally-managed --record record.txt
```

Comment:
I'm happier just leaving it off for now, as it doesn't seem the right time to be doing tests. I'll spend some time and figure out how to do the nose suite testing after the fact later when I get some time. For now this seems like it's reasonable, and I would like to manage to get it in and working in a feedstock if possible rather than waiting for me to figure out how to run what it a fairly incomplete test suite.